### DISCUSSION TITLE
Copilot gets disabled, needs udpdate, can't update

### DISCUSSION THREAD

**Discussion by angel-bbtk** [community/community#18111] [Copilot Conversations] [answered]
Copilot in VSCode gets disabled asking me to upgrade the extension but it does not allow to update to latest 1.27.*
I'm stuck in 1.17.5850

I've tried uninstalling/installing but it installs 1.17 again

**Comment by AmelMechalikhY** [comment]
same problem on my end, any update on this ?

**Comment by davecheney** [comment, marked as answer]
@angel-bbtk @AmelMechalikhY please see https://github.com/orgs/community/discussions/30716

**Reply by AmelMechalikhY** [reply]
thank you @davecheney I updated vscode then I was able to update copilot. Thank you for your reply !
